packages/webview_flutter/lib/webview_flutter.dart

```diff
@@ -308,6 +308,21 @@ class WebViewController {
   }
 }
 
+/// Manages cookies pertaining to all [WebView]s.
+class CookieManager {
+  const CookieManager._();
+
+  static const MethodChannel _channel =
+      MethodChannel('plugins.flutter.io/cookie_manager');
+
+  /// Removes all cookies, and returns true if cookies were
```

Comment:
The first paragraph will be used in the Dartdoc's table of contents. better to keep it short. I'd just have the summary paragraph say "Clears all cookies."
And then in the next paragraph expand on which webview instances are affected, and in a following paragraph describe the return value.